[2025-01-30T00:29:13.347+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: embulk_pipeline_csv.load_csv_to_csv manual__2025-01-30T00:29:09.287030+00:00 [queued]>
[2025-01-30T00:29:13.358+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: embulk_pipeline_csv.load_csv_to_csv manual__2025-01-30T00:29:09.287030+00:00 [queued]>
[2025-01-30T00:29:13.358+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2025-01-30T00:29:13.359+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2025-01-30T00:29:13.359+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2025-01-30T00:29:13.377+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): load_csv_to_csv> on 2025-01-30 00:29:09.287030+00:00
[2025-01-30T00:29:13.384+0000] {standard_task_runner.py:55} INFO - Started process 5645 to run task
[2025-01-30T00:29:13.387+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'embulk_pipeline_csv', 'load_csv_to_csv', 'manual__2025-01-30T00:29:09.287030+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_csv_dag.py', '--cfg-path', '/tmp/tmpj1hx8ruo']
[2025-01-30T00:29:13.388+0000] {standard_task_runner.py:83} INFO - Job 11: Subtask load_csv_to_csv
[2025-01-30T00:29:13.510+0000] {task_command.py:376} INFO - Running <TaskInstance: embulk_pipeline_csv.load_csv_to_csv manual__2025-01-30T00:29:09.287030+00:00 [running]> on host 994f36fe794c
[2025-01-30T00:29:13.644+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=embulk_pipeline_csv
AIRFLOW_CTX_TASK_ID=load_csv_to_csv
AIRFLOW_CTX_EXECUTION_DATE=2025-01-30T00:29:09.287030+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-01-30T00:29:09.287030+00:00
[2025-01-30T00:29:13.645+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2025-01-30T00:29:13.646+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'java -jar /embulk-0.9.25.jar run /opt/***/dags/config_load_csv.yml']
[2025-01-30T00:29:13.659+0000] {subprocess.py:86} INFO - Output:
[2025-01-30T00:29:13.663+0000] {subprocess.py:93} INFO - Error: Unable to access jarfile /embulk-0.9.25.jar
[2025-01-30T00:29:13.664+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-01-30T00:29:13.690+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-01-30T00:29:13.694+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=embulk_pipeline_csv, task_id=load_csv_to_csv, execution_date=20250130T002909, start_date=20250130T002913, end_date=20250130T002913
[2025-01-30T00:29:13.710+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 11 for task load_csv_to_csv (Bash command failed. The command returned a non-zero exit code 1.; 5645)
[2025-01-30T00:29:13.759+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2025-01-30T00:29:13.808+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
